tests/AzureSearchTests.FunctionalTests/AzureIndexFunctionalTests.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using BasicSearchTests.FunctionalTests.Core;
+using Newtonsoft.Json;
+
+namespace AzureSearchTests.FunctionalTests
```

Comment:
Namespace should be similar to the library under test.

Reply:
Fixed